template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/ranger/RangerCloudStorageServiceConfigProvider.java

```diff
@@ -58,6 +65,12 @@
                     .orElseGet(() -> cloudPaths.add(RANGER_AUDIT.name() + "=" + getDefaultRangerAuditUrl(templateProcessor, templatePreparationObject)));
 
 
+            if (datalakeCluster) {
```

Comment:
I agree making this check is a good idea in general, but this config provider is explicitly for Ranger Admin that is supposed to exist in one place only, namely in the DL (or multiple DLs if we ever happen to support that in any environment). See `isConfigurationNeeded()` below, it explicitly checks for the presence of Ranger Admin in the cluster template, so the rest of this config provider (including the encompassing `getServiceConfigs()` method) will not be executed for DH clusters.

In short: I suggest removing the additional check for the cluster kind and let it just assume DL.